### PR TITLE
Show multiple items when initialize

### PR DESCRIPTION
**Pull request by vuquangpham**
## Problem
- We are unable to open multiple sections (`activeSections`) in the array.

## Why
image
We have a condition for blocking  to avoid the double click on the animating item, but we need to escape that condition if we input `activeSections` in the array by default

## Problem
- Add `hasActivated` flag for checking it in `getToggleState` function.

### CODE REVIEW

**Comment by phucbm** [comment]
> We are unable to open multiple sections (activeSections) in the array.

1. Could you link the source code that you're talking about here? And why should we need to be able to "open multiple sections"?

> but we need to escape that condition if we input activeSections in the array by default

2. Why do we need this? 

I don't understand what are you trying to achieve here!

**Comment by phucbm** [comment]
Please merge this branch into branch `2.1.7`.

File: src/_index.js [hunk 1]
```diff
@@ -150,6 +153,8 @@ export class EasyTabAccordion{
             }
         }
 
+        this.hasActivatedSection = true;
```
Comment (phucbm) [inline src/_index.js:156]: What's the point of line 145 when you override the value immediately here?

**Review by phucbm — CHANGES_REQUESTED** [review]
Please check my comments.

**Comment by vuquangpham** [comment]
> > We are unable to open multiple sections (activeSections) in the array.
> 
> 1. Could you link the source code that you're talking about here? And why should we need to be able to "open multiple sections"?
> 
> > but we need to escape that condition if we input activeSections in the array by default
> 
> 2. Why do we need this?
> 
> I don't understand what are you trying to achieve here!

Because we already have these options but we can't use it
image

**Comment by phucbm** [comment]
@vuquangpham I was talking about this one https://github.com/viivue/easy-tab-accordion/pull/40#discussion_r1193293290

**Comment by phucbm** [comment]
@vuquangpham Any updates?

**Comment by vuquangpham** [comment]
@phucbm 
I have renamed the variable. Could you please check it again! Many thanks.

**Review by phucbm — APPROVED** [review]
LGTM.

**Pull request merged**